src/legal-officer/ProtectedUsers.tsx

```diff
@@ -18,17 +22,20 @@ export default function ProtectedUsers() {
                     header: "User",
                     render: request => <Cell content={ `${request.userIdentity.firstName} ${request.userIdentity.lastName}` }/>,
                     align: 'left',
+                    width: "50%",
                     renderDetails: request => <ProtectedUserDetails request={ request } />,
                 },
                 {
                     header: "Other Legal Officer",
                     render: request => <LegalOfficerName address={ request.otherLegalOfficerAddress }/>,
+                    width: "50%",
                     align: 'left',
                 },
                 {
-                    header: "Identity LOC",
-                    render: request => <Cell content={ new UUID(request.decision.locId!).toDecimalString() }/>,
-                    align: 'left',
+                    header: "Action",
+                    render: request => <Button onClick={ () => history.push(locDetailsPath(request.decision.locId!)) }>Identity LOC</Button>,
+                    width: "200px",
```

Comment:
You have then 3 columns of 50% + 50% + 200px, which is above 100%. Or is there some mechanism handling properly the overflow?

Reply:
I agree. The idea is to allocate 200 px for the action column, then share equally the remaining space between the 2 other columns. Not sure how to do that in a clean way.